src/transformState.js

```diff
@@ -6,6 +6,24 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const key of action.keysToRemove) {
+          delete state[key];
+        };
+        break;
+
+      case 'clear':
+        for (const property in state) {
```

Comment:
that's more about key, not property, so I'd give it a name like `key` instead of `property`